bower.json

```diff
@@ -13,6 +13,6 @@
     "web-bootstrap": "./node_modules/meanio/resources/web-bootstrap.js"
   },
   "resolutions": {
-    "angular": "1.4.0-rc.0"
+    "angular": "1.4.8"
```

Comment:
:-D:-)
בתאריך 28 במרץ 2016 23:25,‏ "Lior Kesos" <notifications@github.com> כתב:

> In bower.json
> <https://github.com/linnovate/icu/pull/209#discussion_r57627501>:
>
> > @@ -13,6 +13,6 @@
> >      "web-bootstrap": "./node_modules/meanio/resources/web-bootstrap.js"
> >    },
> >    "resolutions": {
> > -    "angular": "1.4.0-rc.0"
> > +    "angular": "1.4.8"
>
> מי גאון של אמא???!!!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/linnovate/icu/pull/209/files/aefb869e66e2661be3fecaacd256c061e6a34732#r57627501>
>